### PR TITLE
feat: Added Night theme and ISSUE_TEMPLATES

### LINKED ISSUE #65: Having two issue templates to maintain the project
**Actual Behaviour**
The project only has one issue template to 

- report a bug 
- suggest a feature

this it's hard to identify or group issues depending on it's purpose or the content 

**Expected Behaviour**
Since the project is in the development stage it would be great to have two issue templates to maintain issues related to bug reporting or feature requests.

So, when a particular contributor wants to create a issue, once he/she press on the new issue but it will ask/popup the contributor to select the issue type (either bug reporting or feature suggest). Depending on the issue type they can create issues.

below attached is a screenshot of expected issue types

screen shot 2019-03-02 at 2 16 50 am

**Steps to reproduce it**

1. By creating two issue template types to report bugs or suggest features from project settings.

**Would you like to work on the issue?**
Since only the maintainers/contributors of the project has access to create new issue templates, I think one of them have to add it to the project

### PR DESCRIPTION
**Pull request by 0x7FFF**
Fixes #65 #154 

**Changes**: Added ISSUE_TEMPLATES and implemented a Night theme

**Screenshot/s for the changes**: 
P1

**Checklist**:
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [ ] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: 
[night_theme_and_issue_templates.zip](https://github.com/fossasia/neurolab-android/files/3116275/night_theme_and_issue_templates.zip)

### CODE REVIEW

**Comment by ShridharGoel** [comment]
Why did you close the previous PR with the same changes?

**Comment by ShridharGoel** [comment]
And please open separate PRs for separate issues.

**Comment by ShridharGoel** [comment]
Also, there's no open issue for implementing a night theme. First create an issue for the feature.

**Comment by 0x7FFF** [comment]
> Why did you close the previous PR with the same changes?

Codacy didn't accept few line of new code.
Regarding the night theme. Understood, I'll create a new issue and make separate pull requests

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 1]
```diff
@@ -171,4 +194,90 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
+        }
+
+        changeBackgroundColor(colorFromBackground,colorToBackground);
+        changeStatusCardViewColor(colorFromStatusCardView,colorToStatusCardView);
+        changeSetupCardViewColor(colorFromStatusCardView,colorToStatusCardView);
+        changeTextColor(colorFromValues,colorToValues);
+
+    }
+
+    private void changeBackgroundColor(int colorFrom, int colorTo) {
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/MainActivity.java:231]: All the functions are the same, create a single function and pass the view as the third argument

File: app/src/main/java/io/neurolab/MainActivity.java [hunk 2]
```diff
@@ -171,4 +194,90 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
+
+    private void changeTheme() {
+        int colorToBackground;
+        int colorFromBackground;
+        int colorFromStatusCardView;
+        int colorToStatusCardView;
+        int colorFromValues;
+        int colorToValues;
+
+        if (AppCompatDelegate.getDefaultNightMode() == AppCompatDelegate.MODE_NIGHT_YES) {
```
Comment (adityastic) [inline app/src/main/java/io/neurolab/MainActivity.java:206]: Create a enum class

**Review by adityastic — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/neurolab/MainActivity.java (hunk 1 above)
Comment (adityastic) [reply app/src/main/java/io/neurolab/MainActivity.java:231]: Try to make a separate helper class that does the same

Comment (0x7FFF) [reply app/src/main/java/io/neurolab/MainActivity.java:231]: understood, thank you.

**Comment by ShridharGoel** [comment]
@0x7FFF Whenever there are changes required in a PR you can simply create new commits and push them in the same branch. There is no need to close the PR and open a new one for the change.

**Pull request closed without merging**